### PR TITLE
Des 472

### PR DESCRIPTION
**Pull request by mhomolak**
App-header updated to v2.0+ of rebrand

### CODE REVIEW

**Comment by mhomolak** [comment]
@StommePoes @JasonLantz @wyseguyonline @umahaea @udhadpa

**Comment by umahaea** [comment]
Just a note, the build will fail because the selenium tests will fail due to this change.

File: src/scss/dropdown/scss/dropdown-menu.scss [hunk 1]
```diff
@@ -82,7 +103,7 @@
 }
 
 .o-dropdown-menu__heading:not(.o-dropdown-menu__menu-item) {
-	padding: $o-dropdown-menu-menu-item-padding;
+	// padding: $o-dropdown-menu-menu-item-padding;
```
Comment (JasonLantz) [inline src/scss/dropdown/scss/dropdown-menu.scss]: should this be pulled?

Comment (mhomolak) [inline src/scss/dropdown/scss/dropdown-menu.scss]: Yes, good catch.

**Comment by umahaea** [comment]
Tests are written, reviewed. Good to merge..

**Pull request merged**